projects/burnr/src/pages/Home.tsx

```diff
@@ -1,40 +1,55 @@
 import React from 'react';
-import styled from 'styled-components';
 
-import { CardHeader, Card, CardContent, CardMedia, Grid } from '@material-ui/core';
+import { Grid, Paper, Divider, IconButton, Box } from '@material-ui/core';
+import VisibilityIcon from '@material-ui/icons/Visibility';
 
-import { useChainInfo } from '../hooks';
+import { NavTabs, LogoApp, NodeSelector, AccountCard, BalanceValue, NavFooter } from '../components';
 
-interface Props {
-  className?: string;
-}
+function Home ():  React.ReactElement {
 
-function Home ({ className }: Props):  React.ReactElement<Props> {
-	const blockHash = useChainInfo();
+	return (
+		<>
+			<Grid container alignItems='center'>
+				<Grid item xs={6}>
+					<Box paddingX={2}>
+						<LogoApp/>
+					</Box>
+				</Grid>
+				<Grid item xs={6}>
+					<Paper>
+						<Box paddingX={2}>
+							<NodeSelector/>
+						</Box>
+					</Paper>
+				</Grid>
+			</Grid>
+			<Divider/>
+			<Paper>
+				<Box paddingX={2}>
+					<Grid container alignItems='center'>
+						<Grid item xs={6}>
+							<AccountCard
+								account={{
+									address: '13HJwtWXxCfpk8iW9BWg1mBiaMvjUjTxytE8prFkKFiMUztM',
+									name: 'account name',
+								}}
+							/>
+						</Grid>
+						<Grid item xs={6}>
+							<BalanceValue value={1234.56} size='Big' />
+							<IconButton>
+								<VisibilityIcon />
+							</IconButton>
+						</Grid>
+					</Grid>
+				</Box>
+			</Paper>
+			<Divider/>
+			<NavTabs />
+			<NavFooter />
```

Comment:
`<NavFooter>` is also a general App element that shouldn't be specific for single pages. It also belongs in the App.tsx. I'd suggest the following structure:
```
<ThemeProvider theme={appliedTheme}>
	<CssBaseline />
	<ThemeHeader />
	<main className={classes.main}>
		{api && (
			<ApiContext.Provider value={api}>
				<div className='toolbar' />
				<Switch>
					<Route exact path='/' component={Home} />
				</Switch>
			</ApiContext.Provider>
		)}
	</main>
        <NavFooter />
</ThemeProvider>
```